etc/concourse/deploy-pipeline-vars.yml

```diff
@@ -121,6 +121,9 @@ jwtalgo: algo
 # Use templates true/false
 abacus-configure: true
 
+# Slack interval
+slack: 10m
```

Comment:
Makes sense, I changed the default to "5D" to be safe. I used "10m" initially because it seems to be the default when `SLACK` is unset: https://github.com/cloudfoundry-incubator/cf-abacus/blob/a9c064745fe05e1784c0e0f4083f5ff9edb3c116/lib/aggregation/accumulator/src/index.js#L96-L99. Should that be changed too?